Integration portal GA (#930)
amplitude/amplitude-docs: content/collections/partners/en/create-an-event-streaming-integration.md

## Preview and test

> 1. Before  you submit your configuration for review, test the mock payload that you expect to receive from Amplitude from the Testing Tab. The Destination Preview form is identical to what users of the integration will see. Fill in the parameters you defined in this form to use as test values. You can also configure mapped properties and filters on this form. 

Reviewer: ```suggestion
1. Before  you submit your configuration for review, test the mock payload that you expect to receive from Amplitude on the Testing Tab. The Destination Preview form is identical to what users of the integration see. Enter the parameters you defined in this form to use as test values. You can also configure mapped properties and filters on this form. 
```

> 2. Click **Test Connection** to  open the testing modal.  **Test Connection** is available after you enter all required parameters. This enables you to customize your payload. Click **Send Test Event**, any responses from the endpoint you specified appear on the right. 

Reviewer: ```suggestion
2. Click **Test Connection** to  open the testing modal. **Test Connection** is available after you enter all required parameters. This enables you to customize your payload. Click **Send Test Event**, any responses from the endpoint you specified appear on the right. 
```

3. If you see any errors, check the parameters table to make sure all declared parameters are used.
      - **DECLARED:** All declared parameters in the *Create Parameters* section.
      - **USED:** All parameters used either in the URL, headers and event body.

> ## Release Internally

Reviewer: ```suggestion
## Release internally
```

## Preview and test

1. Before submitting your configuration for review, test the mock payload that you expect to receive from Amplitude via the Testing Tab. The Destination Preview form is identical to what users of the integration wil see. Fill in the parameters you defined in this form to use as test values. You can also configure mapped properties and filters on this form. 

> 2. Click the **Test Connection** to generate a open the testing modal. Note that the **Test Connection** button will not be available until all of the required parameters are filled in. This will allow you to customize your payload. Click **Send Test Event**, any responses from the endpoint you specified will appear on the right. 

Reviewer: ```suggestion
2. Click **Test Connection** to  open the testing modal.  **Test Connection** is available after you enter all required parameters. This enables you to customize your payload. Click **Send Test Event**, any responses from the endpoint you specified appear on the right. 
```

3. If you see any errors, check the parameters table to make sure all declared parameters are used.
      - **DECLARED:** All declared parameters in the *Create Parameters* section.
      - **USED:** All parameters used either in the URL, headers and event body.

# Release Internally

> If you would like to test within your org, click **Release Internally**. This will allow anyone within your org to use the integration you defined. Please allow up to 5 minutes for changes to take effect. 

Reviewer: ```suggestion
To test in your org, click **Release Internally**. This enables anyone within your org to use the integration you defined. Allow up to 5 minutes for changes to take effect. 
```
